docs/api/hass_model/hass_model_codegen.md

```diff
@@ -0,0 +1,45 @@
+---
+id: hass_model_codegen
+title: Using the Code generator
+---
+
+NetDeamon has a generator that creates code based on the Entities and services in Home Assitant. This makes it possible to navigate all Entities and Services using intellisense.
+
+The code generator is installed as a global dotNet tool via the following command:
+
+```
+dotnet tool install -g JoySoftware.NetDaemon.HassModel.CodeGen --version 21.46.0-beta
+```
+
+*While in preview, it is required to include the -version of the most recent version explcitily.*
```

Comment:
Maybe include “please check nuget what latest version is” so we don’t have to maintain this.